ui/de-lib/src/main/java/org/iplantc/de/apps/client/presenter/categories/AppCategoriesPresenterImpl.java

```diff
@@ -164,28 +167,41 @@ public void onFailure(Throwable caught) {
             @Override
             public void onSuccess(List<AppCategory> result) {
                 filterCategories(result);
-                createWorkspaceTab(selectedAppCategory);
-                createHPCTab();
+                addCategoriesToWorkspaceTree(selectedAppCategory);
+                addCategoriesToHPCTree();
+                autoSelectHPCCategory();
```

Comment:
Could this be named something like `addHPCTabSelectionHandler`?
I initially thought the HPC category would auto-select every time the apps window was opened 😜